bin/station.js

```diff
@@ -19,24 +19,7 @@ Sentry.init({
   ignoreErrors: [/EACCES/, /EPERM/, /ENOSPC/, /EPIPE/]
 })
 
-const core = new Core(getDefaultRootDirs())
-const modules = [
-  'zinnia',
-  'saturn-L2-node',
-  'bacalhau'
-]
-
-await fs.mkdir(core.paths.moduleLogs, { recursive: true })
-await fs.mkdir(core.paths.metrics, { recursive: true })
-await core.activity.maybeCreateActivityFile()
-await core.metrics.maybeCreateMetricsFile()
-await core.logs.maybeCreateLogFile()
-for (const module of modules) {
-  await fs.mkdir(join(core.paths.moduleCache, module), { recursive: true })
-  await fs.mkdir(join(core.paths.moduleState, module), { recursive: true })
-  await core.metrics.maybeCreateMetricsFile(module)
-  await core.logs.maybeCreateLogFile(module)
-}
+const core = await Core.create(getDefaultRootDirs())
```

Comment:
`Core.create` uses `getDefaultRootDirs` as the default argument value, so maybe we can simplify this line? Unless you want to be explicit about the directories.

```suggestion
const core = await Core.create()
```